GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -275,6 +294,70 @@ private void loadTagInfo(string revision)
             this.InvokeAsync(updateText);
         }
 
+
+        private void loadCommitSignInfo(GitRevision revision)
+        {
+            /* Good commit signature */
+            if (revision.SignInfo == "G")
+            {
+                commitSignPicture.Image = GitUI.Properties.Resources.certificate_ok;
+            }
+            else /* Any kind of not good commit signature */
+            {
+                commitSignPicture.Image = GitUI.Properties.Resources.certificate_error;
+            }
+        }
+
+
+        private int getNumberOfTagHere(GitRevision revision)
+        {
+            int nTagHere = 0;
+
+            foreach (GitRef gitRef in revision.Refs)
+            {
+                if (gitRef.IsTag && gitRef.IsDereference)
+                {
+                    nTagHere++;
+                }
+            }
+
+            return nTagHere;
```

Comment:
the whole method can be inlined with linq to `return revision.Refs.Count(x => x.IsTag && x.IsDereference)`
if the method is used once, there is no need for the method